misk/src/main/kotlin/misk/web/extractors/DefaultParameterExtractorsModule.kt

```diff
@@ -0,0 +1,13 @@
+package misk.web.extractors
+
+import com.google.inject.AbstractModule
+import misk.inject.addMultibinderBinding
+import misk.inject.to
+
+class DefaultParameterExtractorsModule : AbstractModule() {
```

Comment:
Would this ever be used standalone? Why not just put this in WebModule ?

Reply:
Yeah I'll move it to `WebModule`. After talking to @maniksurtani I moved some modules around, and these don't make sense on their own anymore.